src/App.js

```diff
@@ -0,0 +1,22 @@
+import React, {Component} from 'react';
+import * as _ from 'lodash/fp';
```

Comment:
Unused variable.

Please, add `eslint` to project also

Reply:
Done.